sky/sdk/lib/widgets/tool_bar.dart

```diff
@@ -21,7 +21,9 @@ class ToolBar extends Component {
     this.center,
     this.right,
     this.backgroundColor
-  }) : super(key: key);
+  }) : super(key: key) {
+    assert(center != null);
```

Comment:
I'm not sure we want this:
http://www.google.com/design/spec/components/text-fields.html#text-fields-single-line-text-field